build.gradle.kts

```diff
@@ -46,9 +51,24 @@ fun log4j(
     "org.apache.logging.log4j:log4j-$module:2.10.0"
 }
 
-fun webdriver(module: String): String = "org.seleniumhq.selenium:$module:3.11.0"
+fun webdriver(module: String): String = "org.seleniumhq.selenium:$module:$seleniumVersion"
 
 tasks.getByName("wrapper", Wrapper::class).apply {
     gradleVersion = "5.2.1"
     distributionType = Wrapper.DistributionType.ALL
 }
+
+tasks.getByName("test", Test::class).apply {
+    exclude("**/*IT.class")
+}
+
+val testIntegration = task<Test>("testIntegration") {
+    testLogging.exceptionFormat =  org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
+    testLogging.showStackTraces = true
+    testLogging.showExceptions = true
+    testLogging.showCauses = true
+    maxHeapSize = "2G"
+    include("**/*IT.class")
+}
+
+tasks["check"].dependsOn(testIntegration)
```

Comment:
`testIntegration` works locally for me:
```
BUILD SUCCESSFUL in 13m 41s
3 actionable tasks: 3 executed
```

If it doesn't work on CI, then let's remove this `check -> testIntegration` dependency